Dockerfile

```diff
@@ -26,5 +26,5 @@ RUN if [ -z "$EARTHDATA_USERNAME" ] || [ -z "$EARTHDATA_PASSWORD" ]; then \
 # http://www.uvicorn.org/settings/
 ENV HOST 0.0.0.0
 ENV PORT 80
-CMD uv run uvicorn titiler.cmr.main:app --host ${HOST} --port ${PORT} --log-level debug
+CMD uv run uvicorn titiler.cmr.main:app --host ${HOST} --port ${PORT} --log-level debug --reload
```

Comment:
you don't really need the `reload` here because the code is `copied` not mounted

Reply:
I added it because I mount the `titiler/` directory in docker-compose.yml. It is redundant to `COPY` and mount the volume but it is convenient for local development to have the reload functionality.